### ISSUE TITLE
About the missed annotations of OpticCup and Macula

### ISSUE THREAD

**Issue by xuanli98** [LIV4D/MAPLES-DR#1]
Thanks for your great work: MAPLES-DR.  I would be appreciated if you could answer me some questions about  the missed annotation of OpticCup and Macula. In your paper, you mention that "The optic cup is the only exception: In five images, its boundaries were too uncertain for segmentation." However, this is inconsistent with the actual dataset downloaded from [figshare](https://doi.org/10.6084/m9.figshare.24328660)
1) I find that there are 4 annotations missed in the training set of OpticCup (134 labels) and 2 annotations missed in the test set of OpticCup (58 labels), which are 6 annotations missed in total. The filenames of the missed annotations are: ['20060411_60283_0200_PP', '20060411_61060_0200_PP', '20060411_57853_0200_PP', '20060411_59919_0200_PP'] in training set and ['20060412_59636_0200_PP', '20060412_58723_0200_PP'] in test set.
2) There is  one annotation missed in the training set of Macula(137 labels).
The filename of the missed annotation is : ['20051020_55346_0100_PP']

**Comment by gabriel-lepetitaimon** [comment]
Hi, Thanks for your interest in MAPLES-DR! 

We are aware of those missing segmentation maps, actually I removed them myself:
- The 6 missing segmentations of the cup have been removed because the retinologists found the boundaries of the cup too ambiguous for proper annotation. 
- The macula for image `20051020_55346_0100_PP` is not provided because the fundus image is centered on the optic disc and the macula is simply not visible in the image. We could have left an empty segmentation map but prefered to delete it altogether to raise awareness on this issue.

The paper currently available on Arxiv is only a preprint while the real manuscript is under revision. The last version of the manuscript (not published yet) acknowledges those missing biomarkers. In the meantime I've added a warning in the [dataset description](https://liv4d.github.io/MAPLES-DR/en/welcome/dataset_description.html#id7) to prevent future confusion.

On a different note, if you're able to script in python, I strongly encourage you to use the [maples_dr](https://pypi.org/project/maples-dr/) python package to download and format MAPLES-DR. Particularly [maples_dr.configure](https://liv4d.github.io/MAPLES-DR/en/api_reference/quick_api.html#maples_dr.configure) has an option to exclude images that are missing macula and/or optic cup segmentation.
